cpp/dspace/doi.cc

```diff
@@ -0,0 +1,115 @@
+#include <iostream>
+#include <regex>
+#include <set>
+#include <unordered_map>
+#include <cstdlib>
+#include "DbConnection.h"
+#include "DbResultSet.h"
+#include "DbRow.h"
+#include "Downloader.h"
+#include "EmailSender.h"
+#include "HttpHeader.h"
+#include "JSON.h"
+#include "StlHelpers.h"
+#include "StringUtil.h"
+#include "VuFind.h"
+#include "util.h"
+
+using namespace std;
+
+const std::string DSPACE_SERVER_URL("https://publikationen.uni-tuebingen.de/rest/items/");
+const std::string DOI_KEY("dc.relation.uri");
+const std::string PUBLIKATION_TITLE_KEY("dc.title");
+const std::string DOI_URL_FORMAT("http://dx.doi.org/");
+const std::vector<std::string> NOTIFICATION_EMAIL_ADRESSES = { "andrii.lysohor@uni-tuebingen.de" };
```

Comment:
This address should be changed (will be clarified by thefass, maybe even meistertask)